src/Makefile

```diff
@@ -48,7 +48,7 @@ config-gen: update
 	@echo "Configuring the system with clock speed and directory structure."
 	@echo "Please view Makefile.cosconfig and kernel/include/shared/cpu_ghz.h to make sure they're accurate."
 	@echo "Do _not_ 'git add' either of these files."
-	@cat /proc/cpuinfo | grep "model name" -m 1 | sed 's/.*\([0-9]\.[0-9]*\).*/\#define CPU_GHZ \1/' > ./kernel/include/shared/cpu_ghz.h
+	@cat /proc/cpuinfo | grep "cpu MHz" -m 1 | awk '{printf("#define CPU_GHZ %3.2f\n", $$4/1000)}' > ./kernel/include/shared/cpu_ghz.h
```

Comment:
This is great. 
However, I think we can try to get rid of `cpu_ghz.h` entirely?? Just something to think about.